### PR TITLE
modify test_model and fix a bug in create_obj_det_table

### PR DESCRIPTION
**Pull request by wenyushi**
The change includes:
/dlpy/model.py: change print info
/dlpy/model_conversion/write_onnx_model.py a bracket is added
/dlpy/splitting.py: correct a typo
/dlpy/tests/test_model.py: Since deploy save to local path, it should use data_dir_local.
/dlpy/utils.py: I find a better way to create caslib and use it in loadImage. Besides, I specified formatlength of idjoin, so that it solve the issue when parsing name of annotation files with different length.

### CODE REVIEW

File: .pytest_cache/v/cache/nodeids [hunk 1]
```diff
@@ -0,0 +1,56 @@
+[
```
Comment (ethem-kinginthenorth) [inline .pytest_cache/v/cache/nodeids]: please remove these from your commit

Comment (ethem-kinginthenorth) [reply .pytest_cache/v/cache/nodeids]: I added .pytest_cache in gitignore

File: dlpy/utils.py [hunk 2]
```diff
@@ -1101,10 +1101,16 @@ def create_object_detection_table(conn, data_path, coord_type, output,
 
     det_img_table = random_name('DET_IMG')
 
-    caslib, path_after_caslib = caslibify(conn, data_path, task='load')
+    caslib = find_caslib(conn, data_path)
```
Comment (ethem-kinginthenorth) [inline dlpy/utils.py]: I believe I changed this to make sure that if there is already a caslib use it. Why changing back?

Comment (wenyushi) [reply dlpy/utils.py]: It is not changing back. Basically, I need a caslib needs to be created to the data_path folder. After rereading your caslibify function,  I found that set task='save’ solve anything. I will change.

Comment (wenyushi) [reply dlpy/utils.py]: I tried more tests and found both task='load' or task='save' works. Keep this part unchanged.

**Pull request merged**